### ISSUE TITLE
[BUG] Unable to load library 'libSkiaSharp'. Web .NET 4.7.2 - Windows & Azure

### ISSUE THREAD

**Issue by jsgoupil** [mono/SkiaSharp#1481]
**Description**
I'm sadden I have to open this here, but I have been looking for more than 2 hours and I still can't find a solution!
When using a .NET Framework program or .NET Core program, it works great, however, when I use a Web Application that uses the same code, I get the following message. This is the same thing when deploying on Azure (not as an Azure Function, just a WebApp)

```
Unable to load library 'libSkiaSharp'.
at SkiaSharp.LibraryLoader.LoadLocalLibrary[T](String libraryName)
↵   at SkiaSharp.SkiaApi.<>c.<.cctor>b__1723_0()
↵   at System.Lazy`1.CreateValue()
↵   at System.Lazy`1.LazyInitValue()
↵   at System.Lazy`1.get_Value()
↵   at SkiaSharp.SkiaApi.GetSymbol[T](String name)
↵   at SkiaSharp.SkiaApi.sk_managedstream_set_procs(SKManagedStreamDelegates procs)
↵   at SkiaSharp.SKAbstractManagedStream..cctor()"

====

at SkiaSharp.SKAbstractManagedStream..ctor(Boolean owns)
↵   at SkiaSharp.SKManagedStream..ctor(Stream managedStream, Boolean disposeManagedStream)
↵   at SkiaSharp.SKCodec.WrapManagedStream(Stream stream)
↵   at SkiaSharp.SKBitmap.Decode(Stream stream)
```

The /bin folder contains SkiaSharp.dll. I included the Nuget package in the Web csproj. I added `    <ShouldIncludeNativeSkiaSharp>True</ShouldIncludeNativeSkiaSharp>`. Still nothing.

This is not the first time it seems to happen, more than one bug report had been opened about this.
But I see really no easy solution nor documentation about this.
What is the correct way to statically link this or make it work :) ?

**Basic Information**

- Version with issue:  2.80.1
- Last known good version:  ?
- IDE:  Visual Studio
- Platform Target Frameworks: 
  - Windows Classic:  Windows 10
  - Azure: Web App

**Comment by mattleibow** [comment]
I believe this is a duplicate of https://github.com/mono/SkiaSharp/issues/1479 

I did some looking and I think I got a fix: https://github.com/mono/SkiaSharp/pull/1483#pullrequestreview-476991962

Once CI is complete, I'll let you know and then you can try the preview from the preview feed: https://aka.ms/skiasharp-eap/index.json

**Comment by jsgoupil** [comment]
I'll give it a shot as soon as your CI completes.

**Comment by jsgoupil** [comment]
I don't know how long it takes, I don't have access to your build machine. But it takes forever :) I will have to try it tomorrow then.

**Comment by mattleibow** [comment]
@jsgoupil it should be up on the preview feed as v2.80.2-pr.1483.6.

Let me know how it goes and I'll merge and push out a new preview.

**Comment by jsgoupil** [comment]
@mattleibow It works in local web app. But I'm not setup to deploy this on Azure with a private feed. Can you push it to the main as a preview?

**Comment by mattleibow** [comment]
Just pushed out v2.80.2-preview.36 to NuGet.org. Let me know if that works for you.

Apologies for the long delay, CI was _not_ playing nice at all.

**Comment by jsgoupil** [comment]
@mattleibow It works on Azure. Thanks for your work!

**Comment by mattleibow** [comment]
Great to hear! Thanks for the feedback.
